rust/hyperlane-base/src/settings/loader.rs

```diff
@@ -41,19 +24,34 @@ pub(crate) fn load_settings_object<'de, T: Deserialize<'de>, S: AsRef<str>>(
         })
         .collect();
 
-    let builder = Config::builder();
+    let mut base_config_sources = vec![];
+    let mut builder = Config::builder();
+    for env_entry in PathBuf::from("./config")
```

Comment:
nit: Please add a comment, i.e. "Always load default configs"

Reply:
And below, "Load additional user specified configs"

Reply:
Done